### ISSUE TITLE
Controller lights not on when plugged in [BUG]

### ISSUE THREAD

**Issue by Nic1172** [Davidobot/BetterJoy#535]
**Describe the bug**
When running betterjoy with HIDguardian enabled, plugging in a Pro Controller gets recognized by betterjoy, displays battery level, etc. Everything seems to work except the controller itself, which has the 4 green LEDs all turned off and no buttons on the controller do anything. Connecting the controller via bluetooth doesn't provide any errors. (Using version 6.3 of Betterjoy)
![unknown](https://user-images.githubusercontent.com/71569674/93719541-39de1580-fb51-11ea-81e3-02fbfdfda9b7.png)

**Expected behavior**
Expected controller to work while be hidden from Steam.

**Controller (please complete the following information):**
 - Type: Nintendo Switch Pro Controller (Offical) 
 - Connection: USB

**Comment by shukenmg** [comment]
So if you use BT the lights work? and USB the light don't work?

**Comment by Nic1172** [comment]
> So if you use BT the lights work? and USB the light don't work?

Yup. While the lights are off, the controller also doesn't work. While the lights are on, the controller does work. The lights only go on when using BT, even though the controll shows up in the betterjoy UI while using USB

**Comment by Nic1172** [comment]
Any help?

**Comment by shukenmg** [comment]
Try updating windows, uninstall all betterjoy drivers and files, redownload/reinstall the new version from the last 24hrs/
